tests/Unit/Responses/Entities/EntitiesFactory.php

```diff
@@ -82,19 +83,19 @@ protected static function bootUpFactories(): void
                     ? \implode(',', $faker->randomElement([[], ['site', 'robot']]))
                     : $faker->randomElement([[], ['site', 'robot']]),
                 'created_at'  => $as_array
-                    ? DateTimeFactory::toIso8601Zulu($faker->dateTimeThisYear)
-                    : $faker->dateTimeThisYear,
+                    ? DateTimeFactory::toIso8601Zulu(DateTimeImmutable::createFromMutable($faker->dateTimeThisYear))
+                    : DateTimeImmutable::createFromMutable($faker->dateTimeThisYear),
                 'created_by'  => $faker->userName,
                 'updated_at'  => $as_array
-                    ? DateTimeFactory::toIso8601Zulu($faker->dateTimeThisMonth)
-                    : $faker->dateTimeThisMonth,
+                    ? DateTimeFactory::toIso8601Zulu(DateTimeImmutable::createFromMutable($faker->dateTimeThisMonth))
+                    : DateTimeImmutable::createFromMutable($faker->dateTimeThisMonth),
                 'updated_by'  => $faker->userName,
                 'active_from' => $as_array
-                    ? DateTimeFactory::toIso8601Zulu($faker->dateTimeThisMonth)
-                    : $faker->dateTimeThisMonth,
+                    ? DateTimeFactory::toIso8601Zulu(DateTimeImmutable::createFromMutable($faker->dateTimeThisMonth))
+                    : DateTimeImmutable::createFromMutable($faker->dateTimeThisMonth),
                 'active_to'   => $as_array
-                    ? DateTimeFactory::toIso8601Zulu($faker->dateTimeThisMonth)
-                    : $faker->dateTimeThisMonth,
+                    ? DateTimeFactory::toIso8601Zulu(DateTimeImmutable::createFromMutable($faker->dateTimeThisMonth))
+                    : DateTimeImmutable::createFromMutable($faker->dateTimeThisMonth),
```

Comment:
I can get refactoring of this code & similar code below.

But, this is beyond the scope of the task, right?